src/cli/cli.cc

```diff
@@ -2152,16 +2152,41 @@ int main (const int argc, const char* argv[]) {
 
     if (settings.count("build_copy") != 0) {
       fs::path pathInput = settings["build_copy"].size() > 0
-        ? targetPath / settings["build_copy"]
-        : targetPath / "src";
-      fs::copy(
-        pathInput,
-        pathResourcesRelativeToUserBuild,
-        fs::copy_options::update_existing | fs::copy_options::recursive
-      );
+        ? settings["build_copy"]
+        : "src";
+
+      auto paths = split(pathInput.string(), ';');
+      for (const auto& p : paths) {
+        log("p= "+ p);
+        auto mapping = split(p, '=');
+        auto src = targetPath / trim(mapping[0]);
+        auto dst = mapping.size() == 2
+          ? pathResourcesRelativeToUserBuild / trim(mapping[1])
+          : pathResourcesRelativeToUserBuild;
+
+        if (!fs::exists(fs::status(src))) {
+          log("warning: [build] copy entry '" + src.string() +  "' does not exist");
+          continue;
+        }
+
+        if (!fs::exists(fs::status(dst.parent_path()))) {
+          fs::create_directories(dst.parent_path());
+        }
+
+        fs::copy(
+          src,
+          dst,
+          fs::copy_options::update_existing | fs::copy_options::recursive
+        );
+      }
     // @deprecated
-    } else if (settings.count("build_input") != 0 && settings.count("build_script") == 0) {
```

Comment:
it was necessary for backward compatibility. ssc was copying files from `input` only when `script` is not presented

Reply:
actually I think we can remove this `esle if` completely. The feature `input` copy was never documented

Reply:
Going to leave as is as the test suite depended on this before the deprecation